src/Microsoft.Health.Fhir.Core/Properties/AssemblyInfo.cs

```diff
@@ -17,5 +17,6 @@
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.CosmosDb.UnitTests")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Core.UnitTests")]
 [assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Tests.Common")]
-[assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Tests.E2E")]
+[assembly: InternalsVisibleTo("Microsoft.Health.Fhir.Stu3.Tests.E2E")]
```

Comment:
Since the only thing needed are the 3 Smart-on-FHIR related constants. Can we just make the constant class public or just redefine those 3 constants in the test? To make an entire assembly internally visible seems to a little unnecessary?

Reply:
I think making the constant class public makes sense.